Makefile

```diff
@@ -34,10 +34,9 @@ SMF_MANIFESTS_IN =	smf/manifests/muskie.xml.in \
 # Variables
 #
 NAME 			= muskie
-NODE_PREBUILT_TAG       = zone
-NODE_PREBUILT_VERSION	:= v0.10.48
-# sdc-minimal-multiarch-lts 15.4.1
-NODE_PREBUILT_IMAGE     = 18b094b0-eb01-11e5-80c1-175dac7ddf02
+NODE_PREBUILT_TAG       = zone64
+NODE_PREBUILT_VERSION	:= v6.17.0
+NODE_PREBUILT_IMAGE     = c2c31b00-1d60-11e9-9a77-ff9f06554b0f
```

Comment:
This is the sdcnode build on *18.4.0*:

```
        "// image": "minimal-64-lts 18.4.0",
        "image": "c2c31b00-1d60-11e9-9a77-ff9f06554b0f",
```

and not on the 19.4.0 equiv image being used for BASE_IMAGE_UUID below.


Perhaps we should change these to:

```
NODE_PREBUILT_VERSION	:= v6.17.1
#  minimal-64-lts 19.4.0
NODE_PREBUILT_IMAGE     = 5417ab20-3156-11ea-8b19-2b66f5e7a439
```

(there isn't a v6.17.**0** sdcnode build for 19.4.0)

Reply:
I can make that change and test it out.